ui/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/ProcessModelMigrator.scala

```diff
@@ -27,20 +29,25 @@ class ProcessModelMigrator(migrations: ProcessingTypeDataProvider[ProcessMigrati
     for {
       migrations <- migrations.forType(processDetails.processingType)
       displayable <- processDetails.json
-    } yield migrateWithMigrations(migrations, ProcessConverter.fromDisplayable(displayable), processDetails.modelVersion)
+      migrated <- migrateWithMigrations(migrations, ProcessConverter.fromDisplayable(displayable), processDetails.modelVersion)
+    } yield migrated
   }
 
   private def migrateWithMigrations(migrations: ProcessMigrations, process: CanonicalProcess, modelVersion: Option[Int])
-    : MigrationResult = {
+    : Option[MigrationResult] = {
 
     val migrationsToApply = migrations.processMigrations.toList.sortBy(_._1).dropWhile {
       case (migrationNumber, _) => migrationNumber <= modelVersion.getOrElse(0)
     }.map(_._2)
 
-    val resultProcess = migrationsToApply.foldLeft(process) {
-      case (processToConvert, migration) => migration.migrateProcess(processToConvert)
+    if(migrationsToApply.isEmpty) None
```

Comment:
It would look a little bit nicer if we splitted this method to:
- 'findMigrationsToApply' - returning list/option[list]
- `migrateWithMigrations`
But it's minor thing, so feel free to ignore this comment

Reply:
So nicer it is :)